### ISSUE TITLE
learner API: new function predict_trace

### ISSUE THREAD

**Issue by berndbischl** [mlr-org/mlr3tuning#50]
many learners have a "sequence" parameter so you can get full trace from a single training run.

e.g. ntree, boosting rounds, or the "s" param in glmnet

this is should be supported. 

basically we need to allow to "mark up" one hyperparam in the set, with a tag.
and then maybe not have a new predict_trace function but handle this in the normal predict

**Comment by mb706** [comment]
I think the current "predict" tag is actually enough, but we would have to adjust tuning to handle predict-time-only parameters efficiently

**Comment by berndbischl** [comment]
> I think the current "predict" tag is actually enough, but we would have to adjust tuning to handle predict-time-only parameters efficiently

yes, michel and me discussed this a bit. we need an issue in mlr3tuning now to documents the current thoughts
